packages/mermaid/src/diagrams/gantt/parser/gantt.spec.js

```diff
@@ -180,4 +180,33 @@ row2`;
     expect(ganttDb.getAccTitle()).toBe(expectedTitle);
     expect(ganttDb.getAccDescription()).toBe(expectedAccDescription);
   });
+
+  it('should allow for setting the starting weekday to monday for tick intervals', function () {
+    parser.parse('gantt\nweekday monday');
+    expect(ganttDb.getWeekday()).toBe('monday');
+  });
+  it('should allow for setting the starting weekday to tuesday for tick intervals', function () {
+    parser.parse('gantt\nweekday tuesday');
+    expect(ganttDb.getWeekday()).toBe('tuesday');
+  });
+  it('should allow for setting the starting weekday to wednesday for tick intervals', function () {
+    parser.parse('gantt\nweekday wednesday');
+    expect(ganttDb.getWeekday()).toBe('wednesday');
+  });
+  it('should allow for setting the starting weekday to thursday for tick intervals', function () {
+    parser.parse('gantt\nweekday thursday');
+    expect(ganttDb.getWeekday()).toBe('thursday');
+  });
+  it('should allow for setting the starting weekday to friday for tick intervals', function () {
+    parser.parse('gantt\nweekday friday');
+    expect(ganttDb.getWeekday()).toBe('friday');
+  });
+  it('should allow for setting the starting weekday to saturday for tick intervals', function () {
+    parser.parse('gantt\nweekday saturday');
+    expect(ganttDb.getWeekday()).toBe('saturday');
+  });
+  it('should allow for setting the starting weekday to sunday for tick intervals', function () {
+    parser.parse('gantt\nweekday sunday');
+    expect(ganttDb.getWeekday()).toBe('sunday');
+  });
```

Comment:
It might be great to refactor this using `it.each`

Reply:
✅